functest/test_utils.go

```diff
@@ -249,15 +254,17 @@ func getStatus(t testing.TB) statusResponse {
 	return rpcStatusResponse.Result
 }
 
-func activateDaemons(t *testing.T) {
+func activateDaemons(t *testing.T, countDaemon int) {
```

Comment:
actually you can return list of migrationDaemons, so you will iterate over this list instead of iterating `count` times

that might look better

Reply:
Done. If I understood correctly

Reply:
hmm
my idea was :
`func activateDaemons(t *testing.T, countDaemon int) []*User`
so you don't actually need global list of Users. You can create exact ammount for each test

but I'll leave the decision for @megge-dream I'm not sure about code style

Reply:
To explain myself, with 
`func activateDaemons(t *testing.T, countDaemon int) []*User`
you could write tests like this
`
for _, daemon := activateDaemons(t, 3) {
   // do stuff with daemon
}
`